api/src/main/java/io/kafbat/ui/controller/BrokersController.java

```diff
@@ -94,10 +97,24 @@ public Mono<ResponseEntity<Flux<BrokerConfigDTO>>> getBrokerConfig(String cluste
         .operationParams(Map.of(BROKER_ID, id))
         .build();
 
+    Flux<BrokerConfigDTO> brokerConfigs = brokerService.getBrokerConfig(getCluster(clusterName), id)
+        .map(clusterMapper::toBrokerConfig);
+
+    var kafkaCluster = clustersStorage.getClusterByName(clusterName)
```

Comment:
this is redundant, there's a call `getCluster(clusterName)` on line 100 which can be extracted into a var and reused

Reply:
fixed in f803510096598a38e7dbbc0ee7ae7dab314cb7f3